### PR TITLE
add multiple version sqlalchemy testing in tox config

### PR DESCRIPTION
**Pull request by tdamsma**
As a wide range of SQLAlchemy versions is supported, and in preparation of breaking changes in SQLAlchemy 1.4/2.0, I thought it would be a good idea to test exactly whcih versions are supported now. It turns out that tests pass for sqlalchemy 0.9 - 1.3 (depending slightly on the python version, 0.9 and 1.0 are only supported for python <=3.6)

I'll work on supporting SQLAlchemy 1.4 in a separate PR and try to address the issues raised in #54, #57, #58 and #60. But this is at least a nice baseline. 

PS: The setup.py claims minimum supported SQLAlchemy version is 0.7, but the tests break on 0.7

### CODE REVIEW

**Review by icemac — APPROVED** [review]
LGTM, I like the idea.

In #55, the minimum supported version will be set to 0.9.

**Comment by icemac** [comment]
Thank you for your contribution.

According to the contributing policies of the zopefoundation organization you need to sign a contributor agreement before any non-trivial change before it can be merged. For details please consult the [Contributing guidelines for zopefoundation projects](https://github.com/zopefoundation/.github/blob/master/CONTRIBUTING.md).

**Comment by tdamsma** [comment]
> 
> 
> Thank you for your contribution.
> 
> According to the contributing policies of the zopefoundation organization you need to sign a contributor agreement before any non-trivial change can be merged. For details please consult the [Contributing guidelines for zopefoundation projects](https://github.com/zopefoundation/.github/blob/master/CONTRIBUTING.md).

If I knew up front I would need to print and scan stuff I might niot have contributed at all. But now that I have invested in this anyways, I'll churn trough some paperwork to please the our legal overlords

**Comment by tdamsma** [comment]
I am a member now so I can merge this myself, is that the procedure now? I'd rather just present PR's and not have the final responsibility as I am not familiar enough with the Zope project to do so. I just want my SQLAlchemy deprecation warnings to go away and contribute a bit in the process ;)

> 
> 
> LGTM, I like the idea.
> 
> In #55, the minimum supported version will be set to 0.9.

**Comment by jugmac00** [comment]
> I am a member now so I can merge this myself, is that the procedure now?

Yes, you merge this yourself once it has been approved by another developer.

I would not worry too much about responsibility - your changes were approved. When they break something  - well, maybe you get pinged, but nobody can force you to do additional work :-)

**Comment by icemac** [comment]
@tdamsma Thank you for doing the paper work. As this PR is approved the next step would be to merge it. This is usually done by the creator of the PR to prove having signed the contributor agreement.

**Comment by tdamsma** [comment]
Is there a policy to as to use merge commits or squash and merge? Not that it really matters here as this PR is just a single commit, but I if there is a policy I would not want to stray away from it

**Comment by icemac** [comment]
@tdamsma Currently only squash commits are enabled.

**Pull request merged**
